### PR TITLE
Add feature request issue template

### PR DESCRIPTION
**Pull request by sampaiodiego**
## Proposed changes

The idea is to give the community a clear path for raising feature requests or questions.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Hotfix (a major bugfix that has to be merged asap)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+about: Suggest an idea for this project
+
+---
+
+<!--
+
+We use this repository only for bug reports, if you want to open a feature request,
+please do so on the following repository: https://github.com/RocketChat/feature-requests
+
+If there is a feature request already opened make sure to give your thumps up, as we
```
Comment (Sing-Li) [inline .github/ISSUE_TEMPLATE/feature_request.md:12]: Feature request already opened (where?).   

Are you referring to this repo - and suggesting that we should continue to use this repository for Feature Requests already "gotten in"?   

Please either remove this or clarify this statement.

Comment (sampaiodiego) [reply .github/ISSUE_TEMPLATE/feature_request.md:12]: I've said about the other repo on the previous line:

> if you want to open a feature request, please do so on the following repository: https://github.com/RocketChat/feature-requests

maybe having both sentences on the same paragraph helps?

Comment (sampaiodiego) [reply .github/ISSUE_TEMPLATE/feature_request.md:12]: what do you think @Sing-Li ?

**Comment by sampaiodiego** [comment]
closing this one as it looks not helpful anymore

**Pull request closed without merging**